js/bundle_test.go

```diff
@@ -453,7 +453,7 @@ func TestNewBundle(t *testing.T) {
 			require.Len(t, entries, 1)
 			assert.Equal(t, logrus.WarnLevel, entries[0].Level)
 			assert.Contains(t, entries[0].Message, "There were unknown fields")
-			assert.Contains(t, entries[0].Data["error"].(error).Error(), "unknown field \"something\"")
+			assert.Contains(t, entries[0].Data["error"].(error).Error(), "unknown field \"something\"") //nolint:forcetypeassert
```

Comment:
In theory, there is no need to cast to an error here, so the code could be cleaner

```suggestion
			assert.Contains(t, entries[0].Data["error"], "unknown field \"something\"")
```

Reply:
Tested it and it doesn't work I get

>  &errors.errorString{s:"json: unknown field \"something\""} could not be applied builtin len()

Reply:
Weird, before suggesting I've run this locally, but okay :shrug:

Reply:
Does it work for you ?

Reply:
Tried again and yes it didn't work, maybe I run something different, my fault :facepalm:

Reply:
I guess you just forgot to save the file before runnign the test :sweat_smile: